README.md

```diff
@@ -205,47 +205,6 @@ Navigate to the **Executions** tab to view the workflow execution.
 
 Open the Workbench tab and try running the 'greetings' workflow. You will notice that the workflow execution fails. This is because the task_handler.stop_processes() [helloworld.py] function is called and stops all workers included in the app, and therefore, there is no worker up and running to execute the tasks.
```

Comment:
can you update this text as well?
The README should not focus on the hackathon audience but rather general purpose audience.